lib/pf/services/manager/radiusd_child.pm

```diff
@@ -1596,6 +1625,32 @@ sub generate_container_environments {
 
 }
 
+=head2 generate_port
+
+Generate the radius listening port
+
+=cut
+
+sub generate_port {
+    my ($self, $port) = @_;
+    my @eduroam_authentication_source = @{pf::authentication::getAuthenticationSourcesByType('Eduroam')};
+    my $eduport = $eduroam_authentication_source[0]{'auth_listening_port'};
+    $self->{auth_port} = "1812";
+    $self->{acct_port} = "1813";
+    $self->{cli_port} = "1815";
+    $self->{eduroam_port} = $eduport || "11812";
+    $self->{radsec_port} = "2083";
+    if ($cluster_enabled) {
+        $self->{auth_port} = "1822";
```

Comment:
I'd apply the same logic of `$self->{auth_port} + 10` here and in all the ones below to keep this cleaner